cvmfs/catalog_mgr.h

```diff
@@ -183,6 +183,7 @@ class AbstractCatalogManager : public SingleCopy {
   uint64_t GetRevision() const;
   bool GetVolatileFlag() const;
   uint64_t GetTTL() const;
+  bool GetVOMSAuthz(std::string&) const;
```

Comment:
Would different nested catalogs contain different VOMS authentication caveats?  Or is it meant to be fixed per repository?